recipes/stir/meta.yaml

```diff
@@ -0,0 +1,48 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k or win]
+  number: 0
+
+requirements:
+  build:
+    - numpy
+    - cmake
+    - boost
+    - swig
+    - toolchain
+  run:
```

Comment:
You also need `python` as a build+run dep for the package to work properly.

Reply:
Excellent! This change alone made the Windows build possible. Thanks!